howwework.md

```diff
@@ -1,7 +1,9 @@
 # How We Work
 ## Communications
-- Real-time messaging: [BSDLabs Discord server](https://discord.gg/9vZzzxz7y9)
-- Persisted documentation: Primarily on GitHub bsdlabs/ until we need/ find a better place.
```

Comment:
Rephrased to keep it simple for any new member/observer as we haven't thought of any change yet - we'd update it if required anytime.